diff --git a/intel/wsm.c b/intel/wsm.c
index 4cd64dc..4af35e9 100644
--- a/intel/wsm.c
+++ b/intel/wsm.c
@@ -54,7 +54,7 @@ static plat_event_config_t s_wsmep_profiling[COUNT_NUM] = {
	{ PERF_TYPE_RAW, 0x01B7, 0x53, 0x2011, "off_core_response_0" },
	{ PERF_TYPE_HARDWARE, PERF_COUNT_HW_REF_CPU_CYCLES, 0x53, 0, "cpu_clk_unhalted.ref" },
	{ PERF_TYPE_HARDWARE, PERF_COUNT_HW_INSTRUCTIONS, 0x53, 0, "instr_retired.any" },
-	{ PERF_TYPE_RAW, INVALID_CODE_UMASK, 0, 0, "off_core_response_1" }
+	{ PERF_TYPE_RAW, 0x01BB, 0x53, 0x5011, "off_core_response_1" }
 };

 static plat_event_config_t s_wsm_ll = {
